src/app/app.component.ts

```diff
@@ -15,4 +15,8 @@ export class AppComponent {
   onSubmit() {
```

Comment:
Use the **type** features of angular, you can set the return type of some method and you can set the type of some var, for example:

```suggestion
  public name: string
  
  returnName(): string  {
    return this.name
  }
```